services/brig/test/integration/API/Internal.hs

```diff
@@ -370,3 +374,20 @@ getFeatureConfig galley uid = do
 getAllFeatureConfigs :: (MonadIO m, MonadHttp m, HasCallStack) => (Request -> Request) -> UserId -> m ResponseLBS
 getAllFeatureConfigs galley uid = do
   get $ galley . paths ["feature-configs"] . zUser uid
+
+testWritetimeRepresentation :: forall m. TestConstraints m => Opt.Opts -> Manager -> Cass.ClientState -> Brig -> Endpoint -> Galley -> m ()
+testWritetimeRepresentation _ _mgr db brig _brigep _galley = do
+  quid <- userQualifiedId <$> randomUser brig
+  let uid = qUnqualified quid
+
+  ref <- fromJust <$> (runIdentity <$$> Cass.runClient db (C.query1 q1 (C.params C.LocalQuorum (Identity uid))))
+
+  wt <- fromJust <$> (runIdentity <$$> Cass.runClient db (C.query1 q2 (C.params C.LocalQuorum (Identity uid))))
+
+  liftIO $ assertEqual "ts representaiton do not match" ref (writetimeToInt64 wt)
```

Comment:
What's "ts"? There's a typo too in "representaiton".

Reply:
Fixed it in https://github.com/wireapp/wire-server/pull/2994/commits/207971cb567c7a40f9b35d4f027b2d6154d3ab7a